### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by lugard1**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -52,7 +52,6 @@ <h3>Here's a time line of Dr. Borlaug's life:</h3>
     
             -- Indian Prime Minister Manmohan Singh</p>
             <footer>If you have time, you should read more about this incredible human being on his<a href="https://en.wikipedia.org/wiki/Norman_Borlaug">Wikipedia entry</a></footer>
-        </main>
-
+        </main>
```
Comment (lugard1) [inline index.html:55]: Please use appropriately, the HTML5 semantic tags. it makes your code properly arranged and easily read.

Please use two steps indentation. Also it aids readability.

File: index.html [hunk 2]
```diff
@@ -9,9 +9,9 @@
 </head>
 <body>
     <main id="main">
-        <center>
+         <center>
           <h1 id="title">Dr. Norman Borlaug</h1>
-        </center>
+         </center>
```
Comment (lugard1) [inline index.html:14]: Outdated.
Please use appropriately, the HTML5 semantic tags. instead use the styles.css to position your header

File: index.html (hunk 1 above)
Comment (lugard1) [inline index.html:55]: Please use the HTML5 semantic tags properly to arrange your work. It aids readability and enables easy debugging.

**Review by lugard1 — COMMENTED** [review]
# Changes required ♻️ 

Please check my comments below and make all the required changes.

Submit for a re-review when you are done.

Thanks 😀

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hi @lugard1,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Highlights

 - Polite and friendly comments. ✔️
 - Good use of inline comments. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**